### DISCUSSION TITLE
trivy convert needs more detail

### DISCUSSION THREAD

**Discussion by huornlmj** [aquasecurity/trivy#4737] [Documentation]
### Description

Trivy convert is a new feature that is not documented enough.  For example, can I use it to create a JSON list of packages and convert it back into a table of components in table view?

### Suggestions

Enhance the documentation for trivy convert, explain what exactly it can do, and why it is not just inherently part of the other core methods.

**Comment by knqyf263** [comment]
It's documented here.
https://aquasecurity.github.io/trivy/v0.43/docs/configuration/reporting/#converting

>a table of components in table view

It is not supported in the first place. Trivy shows security findings in the table format.

**Reply by LarsPellarin** [reply]
thats a 404 for that link

**Reply by knqyf263** [reply]
https://trivy.dev/docs/v0.68/guide/configuration/reporting/#converting
